src/main/scala/temple/Main.scala

```diff
@@ -26,6 +27,8 @@ object Main extends App {
     case error: FileAlreadyExistsException  => exit(s"File already exists: ${error.getMessage}")
     case error: NoSuchFileException         => exit(s"File not found: ${error.getMessage}")
     case error: InvalidPackageNameException => exit(s"Package name isn't valid: ${error.getMessage}")
+    case error: InvalidTemplefileException  => exit(error.getMessage)
+    case error: IllegalArgumentException    => exit(error.getMessage)
```

Comment:
I know this is going to make the control flow slightly messier, but I feel this `IllegalArgumentException` should be around `new TempleConfig`, as IllegalArgumentException could well be thrown by the rest of the program too